### ISSUE TITLE
`dict.get()` does not take default argument correctly.

### ISSUE THREAD

**Issue by faze-geek** [lcompilers/lpython#1684]
```
def f():
    b:dict[i32,i32] = {1:2}
    print(b.get(3))
    print(b.get(3,100))
f()

(lp) C:\Users\kunni\lpython>python try.py
None                    # When default argument is not given.
100                     # When default argument is given the key 3 is expected to take value 100.

(lp) C:\Users\kunni\lpython>src\bin\lpython try.py
3670136                # Expected since Lpython assigns garbage value for i : i32 declarations. 
3670136                # Unexpected 100 should be assigned.
```

**Comment by faze-geek** [comment]
@Smit-create You added these lines in #435. I suspect the error is generated here so could you have a look. Meanwhile I will also investigate this issue.
https://github.com/lcompilers/lpython/blob/2d25371215d27b0f30aa14c6846e55d28828cba5/src/lpython/semantics/python_attribute_eval.h#L269-L280

**Comment by Smit-create** [comment]
The problem is not with ASR or frontend which I have added in that PR. It's presently not handled in LLVM.

**Comment by faze-geek** [comment]
>  The problem is not with ASR or frontend which I have added in that PR. It's presently not handled in LLVM.

Yes I realized that . Could you point out which files have the get method. I'll try to handle this case in LLVM.

**Comment by Smit-create** [comment]
This is fixed now. The only thing we need to decide is when/how to use `None`.

**Comment by faze-geek** [comment]
> This is fixed now. The only thing we need to decide is when/how to use `None`.

Thanks @Smit-create . I will test the pr locally. As far as `None` is concerned, does it make sense with static typing ? Something like `d = {'a':None,'b':1,'c':2}` isn't supported as of now.
